redbadger/continous-compliance
# Create a GitHub action that print Hello on Dracula

Closes #3 

## How to test
- Go to `count-dracula` project
- Fetch new branches `git fetch`
- Go to branch *test-continous-compliance* `git checkout test-continous-compliance`
- Do an empty commit `git commit --allow-empty -m "Trigger continous-compliance CI/CD"`
- Check Dracula actions [here](https://github.com/redbadger/count-dracula/actions) 
- 🚀 

![sissy-on-dracula](https://user-images.githubusercontent.com/10330222/113889374-14e28480-97bb-11eb-9700-5e2a3d287864.png)

![](https://media.giphy.com/media/3oEduL6l99F6uKoYzS/giphy.gif)
